sbt-akka-serialization-helper/src/main/scala/org/virtuslab/ash/AkkaSerializationHelperPlugin.scala

```diff
@@ -51,7 +54,8 @@ object AkkaSerializationHelperPlugin extends AutoPlugin {
         new File(
           (ashDumpPersistenceSchema / ashDumpPersistenceSchemaOutputDirectoryPath).value) / (ashDumpPersistenceSchema / ashDumpPersistenceSchemaOutputFilename).value,
       ashDumpPersistenceSchema / ashDumpPersistenceSchemaOutputFilename := s"${name.value}-dump-persistence-schema-${version.value}.yaml",
-      ashDumpPersistenceSchema / ashDumpPersistenceSchemaOutputDirectoryPath := ashCompilerPluginCacheDirectory.value.getPath) ++
+      ashDumpPersistenceSchema / ashDumpPersistenceSchemaOutputDirectoryPath := ashCompilerPluginCacheDirectory.value.getPath,
+      ashCodecRegistrationCheckerCompilerPlugin / sourceCodeDirectory := s"--source-code-directory=${baseDirectory.value.getAbsolutePath}${File.separator}src${File.separator}main${File.separator}scala") ++
```

Comment:
Okay... so just this one potentially left to sort out... other than that, PR ready to merge I think :)

Reply:
switched to use `sourceDirectory` - unfortunately - did not find more elegant working solution